Documentation/operator.md

```diff
@@ -43,6 +43,8 @@ Usage of ./operator:
     	Config Reloader memory limits. Value "0" disables it and causes no limit to be configured. (default 50Mi)
   -config-reloader-memory-request value
     	Config Reloader memory requests. Value "0" disables it and causes no request to be configured. (default 50Mi)
+  -controller-id operator.prometheus.io/controller-id
```

Comment:
nit: Was it intentional to add `operator.prometheus.io/controller-id` in here? I think `string` + the description are sufficient?